dotcom-rendering/src/web/components/Page.tsx

```diff
@@ -20,6 +24,10 @@ type Props = {
  * @param {ArticleFormat} format - The format model for the article
  * */
 export const Page = ({ CAPI, NAV, format }: Props) => {
+	const renderCommercialMetrics =
+		!isServer &&
+		window.guardian.config?.ophan !== undefined &&
+		CAPI.config.switches.commercialMetrics;
```

Comment:
Please correct me if I've misunderstood, but is the `Page` component SSR only? Which would mean `renderCommercialMetrics` is only evaluated server side, and therefore always `false`, which means that the commercial metrics island isn't rendered?

Reply:
This is a really good point. I'm going to step back and look at if there are better ways to handle this situation as there could be higher level solutions we want to introduce.

Reply:
Looks great, thanks!